cluster/gce/gci/configure-helper.sh

```diff
@@ -2855,6 +2811,51 @@ spec:
 EOF
 }
 
+function create-apply-addon-manager-rbac {
```

Comment:
Lets do the same thing here as with kubemark (i.e. give it a token that's associate with the `system:masters` RBAC group). I don't think there's a point in trying to prune the permissions of a utility whose raison detre is to `kubectl apply` arbitrary files. Add an issue & TODO for $RESPONSIBLE_PARTY to rip out the token when it isn't needed any more